Grids: add a note about choosing a component to GS tutorials (#8235)
DevExpress/devextreme-documentation: concepts/05 UI Components/CardView/05 Getting Started with CardView/00 Getting Started with CardView.md

#include uiwidgets-grids-choose-a-data-management-component

#include tutorials-intro-installationnote

> CardView is a UI component that enables you to display data in a card-based format. You can use it for employee directories, contact lists, product catalogs, or task boards.

Reviewer: ```suggestion
CardView is a UI component that displays data in a card-based format. You can use it for employee directories, contact lists, product catalogs, and task boards.
```
